pages/1.11/release-notes/1.11.3/index.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.11.3
+title: Release Notes for 1.11.3
+menuWeight: 0
+excerpt: Release notes for DC/OS 1.11.3
+---
+
+DC/OS 1.11.3 was released on June 26, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.3/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.11.3 includes the following:
+
+- Apache Mesos 1.5.2 [change log](https://github.com/mesosphere/mesos/blob/197c9105c2ed08f1529602f1ff72de846cde98c6/CHANGELOG).
+- Marathon 1.6.496 [change log](https://github.com/dcos/dcos/blob/1.11.3/packages/marathon/buildinfo.json).
+- Metronome 0.4.2 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.2).
```

Comment:
This section does not make clear if these were changed compared to 1.11.2. Why did we change the structure that we previously used?